conf/routes

```diff
@@ -27,3 +27,7 @@ POST  /api/live/:atomType/:id                    controllers.App.publishAtom(ato
 
 PATCH /api/preview/:atomType/:id                 controllers.App.updateAtomByPath(atomType: String, id: String)
 PUT   /api/preview/:atomType/:id                 controllers.App.updateEntireAtom(atomType: String, id: String)
+
+
+#Support
+GET /support/previewCapi/*path                   controllers.Support.previewCapiProxy(path: String)
```

Comment:
So we can match /support/previewCapi/atom/cta/1234 if we wanted to (https://playframework.com/documentation/2.5.x/ScalaRouting#Dynamic-parts-spanning-several-/)